draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+{: #iana}
 # IANA Considerations
 
-See Body {{mybody}}.
+This document has a number of IANA considerations, as described in the following subsections.
+In summary, 6 new registries are established with this request, with initial entries provided for each registry.
+New values for 5 other registries are also requested.
+
+{: #iana-cose-header-parameters-registry}
+## COSE Header Parameters Registry
+
+The 'corim metadata' parameter has been added to the "COSE Header Parameters" registry:
+
+* Name: 'corim metadata'
+
+* Label: 11
+
+* Value: corim-meta-map
+
+* Description: Provides a map of additional metadata for a CoRIM payload composed of (1) one or more entities that created or signed the corresponding CoRIM and (2) its period of validity
+
+* Reference: 'corim-meta-map' in {model-corim-meta-map} of this document
+
+{: #iana-corim-map-items}
+## CoRIM Map Items Registry
+
+This document defines a new registry titled "CoRIM Map".
+The registry uses integer values as index values for items in 'unsigned-corim-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-map-items-reg-procedures title="CoRIM Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.id | RFC-AAAA
+| 1 | corim.tags | RFC-AAAA
+| 2 | corim.dependent-rims | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-corim-map-items title="CoRIM Map Items Initial Registrations"}
+
+{: #iana-corim-locator-map-items}
+## CoRIM Locator-Map Items Registry
+
+This document defines a new registry titled "CoRIM Locator Map".
+The registry uses integer values as index values for items in 'corim-locator-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-locator-map-items-reg-procedures title="CoRIM Locator Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Locator Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.href | RFC-AAAA
+| 1 | corim.thumbprint | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-corim-locator-map-items title="CoRIM Locator Map Items Initial Registrations"}
+
+{: #iana-corim-validity-map-items}
+## CoRIM Validity-Map Items Registry
+
+This document defines a new registry titled "CoRIM Validity Map".
```

Comment:
validity-map isn't extensible today. It should not be extensible.

Reply:
flagged for pruning